app/modules/kit-library/assets/js/components/item-header.js

```diff
@@ -156,6 +156,7 @@ export default function ItemHeader( props ) {
 			}
 			{
 				downloadLinkData && <ApplyKitDialog
+					id={ props.model.id }
```

Comment:
where did you add the `id` prop to the model?

Reply:
The model already has an id